### DISCUSSION TITLE
Multiple collections vs Single Collection

### DISCUSSION THREAD

**Discussion by paul-tharun** [qdrant/qdrant#5494] [Q&A] [answered]
Hey, 
Our RAG application has data from multiple sources, with around 7M points. At present we use one collection for each data source. 
We are considering switching to moving to one single collection for all the data and differentiating the source based on a key in payload. 
Would this be the most efficient way resource wise to set up qdrant? 
Will one collection take up less resources than multiple collections with same number of points?

**Comment by timvisee** [comment, marked as answer]
We do usually recommend to use a single collection for multiple tenants.

Please see our documentation on it here: https://qdrant.tech/documentation/guides/multiple-partitions/

>  Would this be the most efficient way resource wise to set up qdrant?

Yes, very likely.

>  Will one collection take up less resources than multiple collections with same number of points?

Yes.
